Update CHANGELOG.md (#4173)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

#### salesforcedx-vscode-core

> - The **SFDX: Rename Component** command now prevents renaming of an LWC or Aura component if the new name breaks any naming rules. For an LWC component, the new component name is also automatically revised  to started with a lower-case letter if needed. ([PR #4145](https://github.com/forcedotcom/salesforcedx-vscode/pull/4145))

Reviewer: ```suggestion
- The **SFDX: Rename Component** command now prevents you from renaming an LWC or Aura component if the new name breaks any naming rules. New LWC component names are automatically revised to start with a lower-case letter if they don't already. ([PR #4145](https://github.com/forcedotcom/salesforcedx-vscode/pull/4145))
```

#### salesforcedx-vscode-lightning
#### salesforcedx-vscode-lwc

> - You can now use **Ctrl+Space** to re-trigger autocomplete within code braces ({}) in an HTML file. Earlier, moving away from, or deleting content within code braces resulted in loss of autocompletion. ([PR #4144](https://github.com/forcedotcom/salesforcedx-vscode/pull/4144))

Reviewer: ```suggestion
- You can now use **Ctrl+Space** to retrigger autocomplete within code braces ({}) in an HTML file. Previously, if you moved away from or deleted content within code braces, you lost autocompletion. ([PR #4144](https://github.com/forcedotcom/salesforcedx-vscode/pull/4144))
```
